src/Components/Home/Home.js

```diff
@@ -14,25 +14,25 @@ import libra from'../../images/libra.png';
 import scorpio from'../../images/scorpio.png';
 import sagittarius from'../../images/sagittarius.png';
 
-const Home = () => {
+const Home = (props) => {
     const images = [
-        {src: capricorn, alt:'Capricorn'},
-        {src: aquarius, alt: 'Aquarius'},
-        {src: pisces, alt: 'Pisces'},
-        {src: aries, alt: 'Aries'},
-        {src: taurus, alt: 'Taurus'},
-        {src: gemini, alt: 'Gemini'}, 
-        {src: cancer, alt: 'Cancer'}, 
-        {src: leo, alt: 'Leo'},
-        {src: virgo, alt: 'Virgo'},
-        {src: libra, alt: 'Libra'},
-        {src: scorpio, alt: 'Scorpio'}, 
-        {src: sagittarius, alt: 'Sagittarius'}
+        {src: capricorn, alt:'capricorn'},
```

Comment:
If you wanted to move this array into a separate data file you definitely could, but its not huge so I dont know if you need to. Just a potential nice to have future refactor!